compiler/rustc_mir_build/src/check_tail_calls.rs

```diff
@@ -63,6 +63,10 @@ impl<'tcx> TailCallCkVisitor<'_, 'tcx> {
                 "`become` outside of functions should have been disallowed by hit_typeck"
```

Comment:
(maybe while you're here)

```suggestion
                "`become` outside of functions should have been disallowed by hir_typeck"
```